common/config/rush/pnpm-lock.yaml

```diff
@@ -9742,16 +9695,16 @@ packages:
       '@azure-tools/test-recorder': 1.0.2
       '@azure/identity': 2.0.4
       '@microsoft/api-extractor': 7.18.11
-      '@rollup/plugin-commonjs': 21.1.0_rollup@2.73.0
-      '@rollup/plugin-json': 4.1.0_rollup@2.73.0
-      '@rollup/plugin-multi-entry': 4.1.0_rollup@2.73.0
-      '@rollup/plugin-node-resolve': 13.3.0_rollup@2.73.0
+      '@rollup/plugin-commonjs': 21.1.0_rollup@2.75.5
+      '@rollup/plugin-json': 4.1.0_rollup@2.75.5
+      '@rollup/plugin-multi-entry': 4.1.0_rollup@2.75.5
+      '@rollup/plugin-node-resolve': 13.3.0_rollup@2.75.5
       cross-env: 7.0.3
       mkdirp: 1.0.4
       mocha: 7.2.0
       rimraf: 3.0.2
-      rollup: 2.73.0
-      rollup-plugin-sourcemaps: 0.6.3_rollup@2.73.0
+      rollup: 2.75.5
+      rollup-plugin-sourcemaps: 0.6.3_rollup@2.75.5
       tslib: 2.4.0
       typescript: 4.2.4
```

Comment:
is that much activity here needed? would most if the changes go away if you checkout this file from the latest main branch and run `rush update`?

Reply:
Not necessary, but I usually do regenerate the whole lock file when there are conflicts in it. Since we update the lock file weekly, automatically anyway, I don't think it makes much of a difference.